app/services/preserved_object_handler.rb

```diff
@@ -161,7 +164,7 @@ def update_version
       Rails.logger.debug "update_version #{druid} called"
       if endpoint.endpoint_type.endpoint_class == 'online'
         # NOTE: we deal with active record transactions in update_online_version, not here
-        update_online_version
+        update_online_version(false, nil, true)
```

Comment:
why is `set_status_to_enexp_version` here true?

Reply:
i don't understand your question - typo?

Reply:
`update_online_version` the method takes in three parameters: `moab_validated`, `status`, and `set_status_to_unexp_version`, here you set it to be ` true`. I was wondering why we are setting it to `true`.

Reply:
my thinking was:  

in the case of update_version, if we don't get  incoming > catalog, then we should set the status to "whoa!  this catalog entry isn't what I expected!"   But maybe that's wrong.  I sort of made it up as I went.

Reply:
if incoming is NOT > catalog for an update_version call (NOT a check_existence call) then the issue is with the "incoming" argument, and the PC status should NOT be updated.

Example:  ingest robot has a new version, so it calls update_version with v3.  The catalog says it is already at v3.  Now what?   Ask Julian / pick something for now.

Reply:
Per Ben:

Note:  if the call is coming from ingest robot, then _we can trust the incoming version_.  If
- incoming version == catalog version + 1:  perfect.  update catalog version.  Joy to the world.
- incoming version == catalog version + (more than 1):  report error to dor workflows.  log error.  But go ahead and update the catalog.
- incoming version == catalog version OR incoming version < catalog version:  report error to dor workflows.  log error.  Do not update.  Do not pass go.

Reply:
I created #451 to capture the above from Ben, rather than addressing it here.